src/ansys/fluent/visualization/plotter/plotter_windows_manager.py

```diff
@@ -100,19 +101,19 @@ def close(self):
 class PlotterWindow(PostWindow):
```

Comment:
@prmukherj We could rename `PostWindow` to `VisualizationWindow` since the latter is (correctly) more general.

Reply:
In the backend these are used, but not user facing from the top-level wrapper.

Reply:
Renaming has been done and makes sense. Thank you.